javascript/javascript1/week3/week3-guzide/warmups/script.js

```diff
@@ -0,0 +1,84 @@
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+let removeElement = names.indexOf(nameToRemove);
```

Comment:
Good job! 😄 As a small nitpick, good naming of variables will only get more important and here some might think the have the element/name itself but it is the index of the name. Can you think of a name that might be less confusing? 🙂

Reply:
More logical variable name is used. Thank you :)